sampleReact.jsx

```
@@ -1,6 +1,7 @@
 function TestComponent() {
     // ruleid:react-dangerouslysetinnerhtml
     return <div dangerouslySetInnerHTML={createMarkup()} />;
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)